<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usuarios</class>
 <widget class="QWidget" name="usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>470</width>
     <height>570</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="bnt_modificar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>110</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/editar_usuario.png</normaloff>../template/black/editar_usuario.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="contenedor">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>110</y>
      <width>460</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="bnt_arriba">
     <property name="geometry">
      <rect>
       <x>387</x>
       <y>34</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_arriba2.png</normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_arriba2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QListWidget" name="list_user">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>35</y>
       <width>381</width>
       <height>418</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="bnt_abajo">
     <property name="geometry">
      <rect>
       <x>387</x>
       <y>380</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_abajo2.png</normaloff>../../../Users/Javier/Documents/pruebas/imagenes/icono_abajo2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bnt_home">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>10</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/icono_home.png</normaloff>../template/black/icono_home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bnt_nuevo">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/nuevo_usuario.png</normaloff>../template/black/nuevo_usuario.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bnt_ayuda">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>10</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/ayuda.png</normaloff>../template/black/ayuda.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="l_logo">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>260</y>
     <width>451</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../logo/logo_empresa.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="fondo_sombra">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_usuario_acceso">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>970</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_fecha_hora">
   <property name="geometry">
    <rect>
     <x>1295</x>
     <y>969</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <zorder>l_logo</zorder>
  <zorder>fondo_sombra</zorder>
  <zorder>frame</zorder>
  <zorder>l_usuario_acceso</zorder>
  <zorder>l_fecha_hora</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
